.circleci/config.yml

```diff
@@ -40,15 +43,22 @@ jobs:
           path: build/reports/tests
 
   integration-test:
-    executor:
-      name: hmpps/java_localstack_postgres_with_db_name
-      jdk_tag: << pipeline.parameters.java-version >>
-      localstack_tag: << pipeline.parameters.localstack-version >>
-      services: "sns,sqs"
-      postgres_tag: "16"
-      postgres_username: "assess-for-early-release-db"
-      postgres_password: "afer"
-      postgres_db: "afer"
+    docker:
```

Comment:
can't currently override PGPORT on the hmpps orb executor so have to specify the the docker images separately for now.. I'll raise a PR with the orb so we can move back